CHANGELOG/CHANGELOG-2.1.md

```diff
@@ -0,0 +1,235 @@
+# Release notes for v2.1.0
+
+[Documentation](https://kubernetes-csi.github.io/docs/)
+# Changelog since v2.0.0
+
+## Changes by Kind
+
+### Feature
+
+- Added --timeout parameter for the unlikely case that a driver's GetPluginInfo call is slower than one second ([#124](https://github.com/kubernetes-csi/node-driver-registrar/pull/124), [@pohly](https://github.com/pohly))
+- `health-port` flag is deprecated and replaced by `http-endpoint`. ([#125](https://github.com/kubernetes-csi/node-driver-registrar/pull/125), [@verult](https://github.com/verult))
+
+### Bug or Regression
+
+- Changing distroless base image from multiarch to -amd64 to fix Prow builds ([#113](https://github.com/kubernetes-csi/node-driver-registrar/pull/113), [@namrata-ibm](https://github.com/namrata-ibm))
```

Comment:
I think we can remove the entry for #113 and #116 since it was only a temporary thing

Reply:
Done